src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriverExtension.java

```diff
@@ -23,7 +23,7 @@
 		extends ExtensionBase
 		implements StorageDriverFactory<I, O, T> {
 
-	private static final String NAME = "pravega";
+	private static final String NAME = "tcp";
```

Comment:
it makes the storage driver name equal to "tcp" which is incorrect
please use the different constants for the URI schema and driver name